functional_tests/artifacts/log-model.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F  # noqa
+import wandb
+from wandb.beta.workflows import log_model
+
+
+class Net(nn.Module):
+    def __init__(self):
+        super(Net, self).__init__()
+        self.conv1 = nn.Conv2d(1, 32, 3, 1)
+        self.conv2 = nn.Conv2d(32, 64, 3, 1)
+        self.dropout1 = nn.Dropout(0.25)
+        self.dropout2 = nn.Dropout(0.5)
+        self.fc1 = nn.Linear(9216, 128)
+        self.fc2 = nn.Linear(128, 10)
+
+    def forward(self, x):
+        x = self.conv1(x)
+        x = F.relu(x)
+        x = self.conv2(x)
+        x = F.relu(x)
+        x = F.max_pool2d(x, 2)
+        x = self.dropout1(x)
+        x = torch.flatten(x, 1)
+        x = self.fc1(x)
+        x = F.relu(x)
+        x = self.dropout2(x)
+        x = self.fc2(x)
+        output = F.log_softmax(x, dim=1)
+        return output
+
+
+def main():
+    my_model = Net()
+
+    _ = log_model(my_model, "my-model", aliases=["boom"], scope_project=True)
```

Comment:
ah interesting... it creates a run implicitly?

Reply:
Yep, if `wandb.run` doesn't exist, we'll create a run for them. We definitely want to encourage users to create a run in the examples for this API, but include implicit run creation for user convenience.